assets/home/tasks-section/tasks.js

```diff
@@ -25,11 +28,14 @@ const Tasks = () => (
 		</p>
 
 		<ul className="sensei-home-tasks__list">
-			<TaskItem label="Set up Course Site" href="#" completed />
-			<TaskItem label="Create your first Course" href="#" completed />
-			<TaskItem label="Create your first Course" href="#" />
-			<TaskItem label="Configure Learning Mode" href="#" />
-			<TaskItem label="Publish your first Course" href="#" />
+			{ items.map( ( task ) => (
+				<TaskItem
+					key={ task.id }
+					label={ task.title }
+					href={ task.url }
+					completed={ task.done }
```

Comment:
hmm, could we use the same props names as coming from the API?

I'm not sure if there's a huge value in remapping the attributes to different names, although this isn't a big blocker.

Reply:
Sure! Changed here: https://github.com/Automattic/sensei/pull/5902/commits/c11d523e998a3a10a5626e51e530e9252289880b